packages/office-ui-fabric-react/src/components/Autofill/Autofill.types.ts

```diff
@@ -83,8 +83,6 @@ export interface IAutofillProps extends React.InputHTMLAttributes<HTMLInputEleme
    * Handler for checking and updating the value if needed
    * in componentWillReceiveProps
    *
-   * @param defaultVisibleValue - The defaultVisibleValue that got passed
```

Comment:
Why was this removed, is it not being used anywhere?

Reply:
It wasn't passed into the method, so it couldn't have been used.

Reply:
Strange... is that a regression that happened at some point or was it added and just never hooked up?

Reply:
No idea, but it's been that way for 2 years :/

Reply:
Hmm... maybe a follow up investigation... I wonder if there's potentially other things that might be lurking here potentially due to the fact that this wasn't being used anywhere...

Reply:
This whole prop is going to become a problem soon because `componentWillReceiveProps` is going away...also it's generally a bit "smelly" and makes me wonder if there's a better way to achieve whatever this was meant to achieve.

Reply:
There definitely is a better way to do this, we should refactor it so that the autofill can be a controlled component, rather than this weird callback way of handling it. All it would need to do is take in a value.

Reply:
Oh gosh, *that's* what this callback was doing...??? 🤦‍♀ 🤦‍♀ 🤦‍♀

Reply:
Sure seems like that's what it was doing.